backend/app/worker.py

```diff
@@ -190,3 +190,32 @@ def send_email(email):
 @dramatiq.actor
 def send_one_email(message: str, dest: str):
     send_one_email_impl(message, dest)
+
+
+@dramatiq.actor
+def update_quality_moderation():
+    with WorkerDB() as sqldb:
+        appids = {app[5:] for app in db.redis_conn.smembers("apps:index")}
+
+        if not appids:
+            return
+
+        appids_for_frontend = [
+            appid for appid in appids if appid and db.is_appid_for_frontend(appid)
+        ]
+
+        if not appids_for_frontend:
+            return
+
+        for appid in appids_for_frontend:
+            # Check app name lenght
+            if value := db.get_json_key(f"apps:{appid}"):
+                models.QualityModeration.upsert(
+                    sqldb, appid, 8, len(value["name"]) <= 20, None
```

Comment:
I'm not a huge fan of using "magic numbers" for the guideline IDs. Strings would probably be clearer.

Another thing to consider is it might be easier to store the guidelines in code within the backend (or even the frontend), rather than in the database.

Reply:
I think I had a reason for this, but I can't remember.

I'm also not sure, if moving stuff into code (backend) would improve. I guess it's easier, as we have one deployed system, but this code is ready and this makes it arguably easier to extend.